setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = visionmetrics
+version = 0.0.1
+description = Evaluation metric codes for various vision tasks.
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Ping Jin, Shohei Ono
+license = MIT
+url = https://github.com/microsoft/vision-evaluation
```

Comment:
We might want to make a new repository. If we go with  "visionmetrics", create microsoft/visionmetrics.